docs/maroon-node.md

```diff
@@ -1,10 +1,15 @@
 # Maroon node
 
-core piece of maroon engine
+Core piece of the maroon engine.
 
+## Epoch publishing
 
+There is no permanent leader. Nodes coordinate via a ring-based time-slice scheduler and commit epochs to etcd with CAS. See [epoch publisher scheduling](./leader-election.md).
 
-## Leader election
-
-through etcd lease
+MN responsibilities
```

Comment:
Let's use `Maroon` for now? Or at least `MRN` if you want an acronym.

Just my two cents.